content/en/docs/instrumentation/cpp/getting-started-new.md

```diff
@@ -0,0 +1,294 @@
+---
+title: Getting Started
+description: Get telemetry for your app in less than 5 minutes!
+cSpell:ignore: rolldice
+weight: 10
+---
+
+This page will show you how to get started with OpenTelemetry in C++.
+
+You will learn how you can instrument a simple C++ application automatically, in
+such a way that [traces](/docs/concepts/signals/traces/), [metrics](/docs/concepts/signals/metrics/) and [logs](/docs/concepts/signals/logs/) are emitted to the console.
+
+## Prerequisites 
+
+Ensure that you have the following installed locally:
+
+- Git
+- C++ compiler supporting C++ version >= 11
+- Make 
+- CMake version >= 3.1
+- [Conan](https://conan.io/)
+
+## Example Application 
+
+The following example uses a basic [Oat++](https://oatpp.io/) application. If you are not using 
+Oat++, that's OK - you can use OpenTelemetry C++ with other web frameworks 
+as well, such as Crow and POCO. For a complete list of 
+libraries for supported frameworks, see the [registry](/ecosystem/registry/?language=cpp).  
```

Comment:
```suggestion
The following example uses a basic [Oat++](https://oatpp.io/) application. If you are not using 
Oat++, that's OK - you can use OpenTelemetry C++ with you can use OpenTelemetry C++ with any C++ application
```

For C++ the link to the registry will not lead the reader to something meaningful, let's keep it simple here (copied from Swift)